dev-python/types-psutil/types-psutil-5.9.5.16.ebuild

```
@@ -0,0 +1,17 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYPI_NO_NORMALIZE=1
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Typing stubs for psutil"
+HOMEPAGE="https://pypi.org/project/types-psutil"
```

```suggestion
HOMEPAGE="https://pypi.org/project/types-psutil/"
```

PyPI always redirects to trailing-slash variants.